frontend/src/styles.css

```
@@ -435,29 +481,31 @@ h3 {
+  height: var(--app-brand-logo-size);
+  border-radius: var(--app-brand-logo-radius);
   object-fit: cover;
   border: none;
 }
 
+:root[data-theme="dark"] .app-brand img {
+  filter: brightness(1.16) saturate(1.08);
+}
+
 .app-brand-text {
   font-family: "Sora", ui-sans-serif, system-ui, sans-serif;
```

Stylelint (17.6.0)

[error] Unexpected quotes around "Sora" (font-family-name-quotes)

(font-family-name-quotes)